.github/workflows/nearcore_deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Nearcore release
+
+on:
+  push:
+    branches: master
+
+  workflow_dispatch:
+    inputs:
+      branch:
+        default: 'master'
+        description: "Nearcore branch to build and publish"
+        type: string
+        required: true
+
+jobs:
+  binary-release:
+    name: "Build and publish neard binary"
+    runs-on: "ubuntu-22.04-8core"
+    environment: deploy
+    permissions:
+      id-token: write # required to use OIDC authentication
+
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::754641474505:role/GitHubActionsRunner
+          aws-region: us-west-1
+
+      - name: Checkout ${{ github.event.inputs.branch }} branch
+        if: ${{ github.event_name == 'workflow_dispatch'}}
+        uses: actions/checkout@v4
+        with:
+          repository: near/nearcore
```

Comment:
Won’t this make it difficult to produce builds from e.g. nearcore-private repository without pushing the code to `near/nearcore` if one wanted to do so?

Reply:
yep, removing it in https://github.com/near/nearcore/pull/10157.
Although, this WF is not intended to be ran on private